### PR TITLE
metrics: Add iperf3 parallel measurements

### LINKED ISSUE #4899: metrics: Add iperf3 parallel measurements

### PR DESCRIPTION
**Pull request by GabyCT**
This PR adds iperf3 parallel measurements which measures the
bandwidth of simultaneos connections make to the server.

Fixes #4899

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: metrics/network/iperf3_kubernetes/k8s-network-metrics-iperf3.sh [hunk 1]
```diff
@@ -116,6 +120,31 @@ EOF
 	fi
 }
 
+function iperf3_parallel() {
```
Comment (chavafg) [inline metrics/network/iperf3_kubernetes/k8s-network-metrics-iperf3.sh:123]: just for better understanding to the user, should we add a comment that the test will use 4 parallel connections?

**Review by chavafg — COMMENTED** [review]
small typo in the commit message: s/simultaneos/simultaneous/

**Comment by GabyCT** [comment]
@chavafg changes applied

**Review by dborquez — APPROVED** [review]

**Review by chavafg — APPROVED** [review]

**Pull request merged**